src/components/ArtistComma.jsx

```diff
@@ -0,0 +1,11 @@
+//Component for comma and &
```

Comment:
Great idea to make this part its own component and import it to the ArtistName component which make the component look much cleaner in the end :)